spec/features/users_appointments_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.feature 'Users::Appointments', type: :feature do
+  before do
+    create_cpip_user_and_login
+  end
+
+  describe 'index' do
+    before do
+      place = create :place, organization: @user.organization
```

Comment:
Préférence perso mais j'aime bien utiliser des `let` pour la création des instances que je mentionne plus loin. Je trouve ça plus lisible qu'un gros block before qui peut parfois contenir plus que de la création de variables. cf: https://relishapp.com/rspec/rspec-core/docs/helper-methods/let-and-let

Reply:
yes bien vu, je le note pour une éventuelle refacto

Reply:
Je pense que là on est vraiment là sur du style, mais comme il n'y a pas d'autre tests, ce qu'il y a dans le before, c'est en fait le début de ton test, tu as coupé ton test en deux artificiellement.